modules/ipi-install-configuring-the-install-config-file.adoc

```diff
@@ -117,7 +117,7 @@ ironic-inspector inspection failed: No disks satisfied root device hints
 
 [NOTE]
 ====
-Before {product-title} 4.12, the cluster installation program only accepted an IPv4 address or and IPv6 address for the `apiVIP` and `ingressVIP` configuration settings. In {product-title} 4.12 and later, these configuration settings are deprecated. Instead, use a list format in the `apiVIPs` and `ingressVIPs` configuration settings to specify IPv4 addresses, IPv6 addresses or both IP address formats.
+Before {product-title} 4.12, the cluster installation program only accepted an IPv4 address or an IPv6 address for the `apiVIP` and `ingressVIP` configuration settings. In {product-title} 4.12 and later, these configuration settings are deprecated. Instead, use a list format in the `apiVIPs` and `ingressVIPs` configuration settings to specify IPv4 addresses, IPv6 addresses or both IP address formats.
```

Comment:
Suggestion to add a comma after 'IPv6 addresses' to comply with comma use for items in a series style mentioned in the [IBM Style Guide](https://www.ibm.com/docs/en/ibm-style?topic=punctuation-commas#commas-between-items-in-a-series). (I know you didn't change this in the PR, but I thought I'd mention it anyway.)

Reply:
Updated As Suggested.